app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+from abc import ABC
+
+
+class Dwarf(Player, ABC):
+    def __init__(self, nickname: str, favourite_dish: str) -> None:
+        super(Dwarf, self).__init__(nickname)
```

Comment:
After the last updates, there is no need to pass Dwarf, self to super(). It will take them automatically

Reply:
fix everywhere